articles/kotlin-ranges/index.md

```diff
@@ -0,0 +1,80 @@
+### Kotlin Range Operators
+Kotlin provides a wide range of solutions. Most of these solutions aim to make development easier and faster. It provides easier ways to perform specific actions. Since it's compiled the same way as Java, its functions and methods ensure your code is more efficient. The boilerplate code present in Java is also greatly reduced. This article addresses one of the solutions it offers in iterations. It will go through Kotlin ranges.
```

Comment:
> It will go through Kotlin ranges.

**In this article, let us take a look at Kotlin ranges** 
would flow better

Reply:
Done with this